engine/src/main/java/com/redhat/cloud/notifications/processors/webhooks/WebhookTypeProcessor.java

```diff
@@ -71,6 +75,8 @@ private void process(Event event, Endpoint endpoint) {
         WebhookProperties properties = endpoint.getProperties(WebhookProperties.class);
 
         final JsonObject payload = transformer.toJsonObject(event);
+        insightsUrlsBuilder.buildInventoryUrl(payload, endpoint.getType().name()).ifPresent(url -> payload.put("inventory_url", url));
```

Comment:
Did you validated with PM we whant to add a new property named "application_url" on webhook (also used by EDA)?

Reply:
Not specifically, this was to maintain consistency with the formatting of other classes that now include the field (PagerDuty and ServiceNow). I'll double-check with this one.

Reply:
Removed based on BU feedback